### ISSUE TITLE
infracost output loses version when combining multiple breakdowns

### ISSUE THREAD

**Issue by mattlqx** [infracost/infracost#511]
When combining multiple json outputs from `infracost breakdown` with `infracost output`, the resulting json output loses the version and is just an empty string. This is infracost 0.8.1.

```
➜ infracost breakdown --path plan-a.json --format json --log-level error > infracost-a.json

➜ infracost breakdown --path plan-b.json --format json --log-level error > infracost-b.json

➜ jq . infracost-a.json  | head
{
  "version": "0.1",
  "resources": [
    {
      "name": "aws_cloudwatch_metric_alarm.api-https-unhealthy-host",
      "metadata": {},
      "hourlyCost": "0.00013698630136986",
      "monthlyCost": "0.1",
      "costComponents": [
        {

➜ jq . infracost-b.json  | head
{
  "version": "0.1",
  "resources": [
    {
      "name": "aws_cloudwatch_metric_alarm.api-https-unhealthy-host",
      "metadata": {},
      "hourlyCost": "0.00013698630136986",
      "monthlyCost": "0.1",
      "costComponents": [
        {

➜ infracost output --format json --path infracost-a.json --path infracost-b.json > infracost_combined.json

➜ jq . infracost_combined.json| head
{
  "version": "",
  "resources": [
    {
      "name": "aws_cloudwatch_metric_alarm.api-https-unhealthy-host",
      "metadata": {
        "filename": "infracost-b.json"
      },
      "hourlyCost": "0.00013698630136986",
      "monthlyCost": "0.1",

```

**Comment by aliscott** [comment]
@mattlqx good find and thanks for the reproduction steps 😄! Have added a fix in https://github.com/infracost/infracost/pull/512 so this will be released in 0.8.2.

**Comment by aliscott** [comment]
@mattlqx thanks for reporting this! The fix has now been released in v0.8.2.
